i18n/locales/en.ts

```diff
@@ -1226,11 +1266,26 @@ const translations = {
         modular_inverse_paragraph_one:
           "The modular inverse of a number a modulo m is a number b such that (a * b) % m = 1. Finding the modular inverse is a critical step in ECDSA signature verification. It's used in the calculation of u1 and u2 during the verification process.",
       },
-      validate_signature: {
+      validate_signature_one: {
         message_verification_heading: 'Importance of Message Verification',
         message_verification_paragraph_one:
           'Message verification enhances the security of communications within the Bitcoin ecosystem. It allows parties to verify the authenticity and integrity of messages, which is valuable in situations where trust and verification are essential, such as in peer-to-peer transactions or communication between parties in a smart contract. Furthermore, message verification serves as a foundation for various applications, including identity verification and the attestation of ownership of a particular Bitcoin address. It adds a layer of cryptographic assurance, reinforcing the trustless and decentralized nature of the Bitcoin network.',
       },
+      validate_signature_two: {
+        base64_encoding_heading: 'Base64 Encoding',
+        base64_encoding_paragraph_one:
+          'Base64 is a basic byte to text encoding scheme that just allows for the data to be convereted to bytes which can then be used in buffers or from bytes to text in such a way that it avoids any problems with url paths and query params. This differs from the baese58 encoding scheme in that it still includes the similar characters that may otherwise be confusing to a user when trying to copy or dicate the encoded message.',
+      },
+      validate_signature_three: {
+        signing_and_ownership_heading: 'Message Signing and Ownership',
+        signing_and_ownership_paragraph_one:
+          "Based on how Bitcoin's psuedo-anonymity we are limited in what we are able to prove because someone can refuse to sign a message even if they do infact have onwership of the correct key to sign a message. However, given this particular key we are able to prove that this key and the owner of said key is not the same person as Satoshi in this case because of the fact that this key does not properly sign the message.",
```

Comment:
```suggestion
          "Based on Bitcoin's pseudo-anonymity we are limited in what we are able to prove because someone can refuse to sign a message even if they do in fact have ownership of the correct key to sign a message. However, given this particular key we are able to prove that this key and the owner of said key is not the same person as Satoshi  because this key does not properly sign the message.",
```

Reply:
Hm yeah the language here is a little funny. Satoshi could have created the invalid signature but it wouldn't prove anything. That's all we can know for sure: this invalid signature is NOT PROOF that whoever created it is Satoshi